client/blocks/inline-help/index.jsx

```diff
@@ -107,101 +104,34 @@ class InlineHelp extends Component {
 		this.inlineHelpToggle = node;
 	};
 
-	moreHelpClicked = () => {
-		this.closeInlineHelp();
-		this.props.recordTracksEvent( 'calypso_inlinehelp_morehelp_click' );
-	};
-
-	toggleContactForm = () => {
-		if ( this.state.showContactForm ) {
-			this.props.recordTracksEvent( 'calypso_inlinehelp_contact_hide' );
-		} else {
-			this.props.recordTracksEvent( 'calypso_inlinehelp_contact_show' );
-		}
-		this.setState( { showContactForm: ! this.state.showContactForm } );
-	};
-
 	render() {
 		const { translate } = this.props;
-		const { showContactForm, showInlineHelp } = this.state;
+		const { showInlineHelp } = this.state;
 		const inlineHelpButtonClasses = { 'is-active': showInlineHelp };
-		const popoverClasses = { 'is-help-active': showContactForm };
-		const showContactButton = abtest( 'inlineHelpWithContactForm' ) === 'inlinecontact';
 		return (
 			<Button
 				className={ classNames( 'inline-help', inlineHelpButtonClasses ) }
 				onClick={ this.handleHelpButtonClicked }
+				onTouchStart={ this.preload }
```

Comment:
i've never seen onTouchStart before. very cool

Reply:
I copy&pasted this from another Calypso component that does section preloading. Not invented by me :) To be 100% correct, there should be also `onFocus={ this.preload }` handler, for keyboard navigation. ESLint rules from the `jsx-a11y` package report warnings for some issues like this.

Reply:
I checked keyboard navigation and it seemed to work well.